src/test_fixture/world.rs

```diff
@@ -63,19 +76,38 @@ impl Default for TestWorldConfig {
                     batch_count: 40,
                 },
             },
+            /// Disable metrics by default because `logging` only enables `Level::INFO` spans.
+            /// Can be overridden by setting `RUST_LOG` environment variable to match this level.
+            metrics_level: Level::DEBUG,
         }
     }
 }
 
+impl TestWorldConfig {
+    pub fn enable_metrics(&mut self) -> &mut Self {
+        self.metrics_level = Level::INFO;
+        self
+    }
+}
+
 impl TestWorld {
     /// Creates a new `TestWorld` instance using the provided `config`.
     #[must_use]
     #[allow(clippy::missing_panics_doc)]
     pub fn new_with(query_id: QueryId, config: TestWorldConfig) -> TestWorld {
+        // setup logging
         logging::setup();
 
+        // setup metrics
+        let metrics_handle = MetricsHandle::new(config.metrics_level);
+
```

Comment:
I prefer not to have comments that say what is being done when it is obvious like this.

Reply:
I prefer it too, but sometimes I forget about it - my bad.